app/controllers/submit.js

```diff
@@ -359,6 +415,74 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
         return this.get('isAdmin') && !(this.get('node.registration'));
     }),
 
+    ///////////////////////////////////////////////////
+    // Language about submission and moderation.
+    ////////////////////////////////////////////////////
+
+    moderationType: Ember.computed('theme.provider.reviewsWorkflow', function() {
+        return this.get('theme.provider.reviewsWorkflow') ?
+            this.get('theme.provider.reviewsWorkflow').toLowerCase() :
```

Comment:
Quatto, `moderationType` could just be `alias('theme.provider.reviewsWorkflow')`